flexmeasures_openweathermap/cli/commands.py

```diff
@@ -95,12 +95,18 @@ def add_weather_sensor(**args):
 @click.option(
     "--location",
     type=str,
-    required=True,
+    required=False,
     help='Measurement location(s). "latitude,longitude" or "top-left-latitude,top-left-longitude:'
     'bottom-right-latitude,bottom-right-longitude." The first format defines one location to measure.'
     " The second format defines a region of interest with several (>=4) locations"
     ' (see also the "method" and "num_cells" parameters for details on how to use this feature).',
 )
+@click.option(
+    "--asset-id",
+    type=int,
+    required=False,
+    help='Get location "latitude,longitude" for a given asset',
```

Comment:
help="ID of a weather station asset - forecasts will be gotten for its location. If present, --location will be ignored."